doc/publishing-api-syntactic-usage.md

```diff
@@ -137,6 +137,14 @@ TODO: Request/Response detail
 Note that link sets can be created before or after the PUT requests for the content item.
 No downstream requests will be sent if the content item doesn't exist yet.
 
+To remove a field from the links update the relevant field with an empty array. For example:
```

Comment:
I am not sure? `link_type` is how we name it internally. I thought it was ubiquitous. I will ask someone who knows.